.github/workflows/pull-request-Dynamo.yml

```diff
@@ -0,0 +1,102 @@
+name: Pull Request CI - Dynamo Infra
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" or "develop" branch
```

Comment:
should the comment mention the "develop" branch?

Reply:
PR should only merge into main - not sure why develop is here?